### PR TITLE
CONTENT-91 - add path param for put/patch

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
this doesn't actually fix the problem yet, it just verifies the validator will catch the problem

File: .github/workflows/content-sources-actions.yml [hunk 1]
```diff
@@ -24,6 +24,16 @@ jobs:
       - run: |
           git diff --exit-code api/openapi.json
 
+  openapivalidate:
+    name: openapi validate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: openapi-generators/openapitools-generator-action@v1
```
Comment (mshriver) [inline .github/workflows/content-sources-actions.yml:32]: Thanks!

**Review by mshriver — APPROVED** [review]
Pulled this into QE environment and was able to generate a client without failing validation.

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/14696935

**Comment by jlsherrill** [comment]
⚠️ This task currently requires qe-approval, but this PR does not fully resolve the task.  Please contact QE to determine appropriate testing required.

**Review by swadeley — COMMENTED** [review]
ACK

**Pull request merged**